### PR TITLE
Fix typos in optwin.vim

### PR DESCRIPTION
**Pull request by k-takata**

### CODE REVIEW

File: runtime/optwin.vim [hunk 1]
```diff
@@ -617,7 +617,7 @@ call <SID>AddOption("mousemodel", gettext("\"extend\", \"popup\" or \"popup_setp
 call <SID>OptionG("mousem", &mousem)
 call <SID>AddOption("mousetime", gettext("maximum time in msec to recognize a double-click"))
 call append("$", " \tset mouset=" . &mouset)
-call <SID>AddOption("ttymouse", gettext("\"xterm\", \"xterm2\", \"dec\" or \"netterm\"; type of mouse"))
+call <SID>AddOption("ttymouse", gettext("\"xterm\", \"xterm2\", \"dec\", \"sgr\", etc.; type of mouse"))
```
Comment (k-takata) [inline runtime/optwin.vim:620]: I think that "sgr" should be the best nowadays if the terminal supports it.

File: runtime/optwin.vim [hunk 2]
```diff
@@ -938,7 +938,7 @@ endif
 
 if has("folding")
   call <SID>Header(gettext("folding"))
-  call <SID>AddOption("foldenable", gettext("set to display all folds open"))
+  call <SID>AddOption("foldenable", gettext("unset to display all folds open"))
```
Comment (k-takata) [inline runtime/optwin.vim:941]: I'm not confident about this, but the meaning looks opposite.

**Comment by k-takata** [comment]
Other thoughts:

* Support for RISC OS has been removed. Do we still need the description of the 'osfiletype' option?
* Support for Farsi has been removed. Do we still need the description of 'altkeymap' and 'fkmap'?

**Comment by k-takata** [comment]
One more comment.
The 'ts' option is set to 15 in this line:
https://github.com/vim/vim/blob/b52d4b02580d20c0241c1f93d0ba47e157777980/runtime/optwin.vim#L150
However there are some options which name is longer than 15 chars:
toolbariconsize, balloonevalterm, quickfixtextfunc and pythonthreehome.

These options are not shown nicely in the optwin window.
How about using the vartabstop option? E.g. `setlocal vts=15,2`
(Or we can increase the value of 'ts' to 17, but it might break the layout of other part.)

**Comment by brammool** [comment]
I'll include the suggestions, thanks.

**Comment by brammool** [comment]
Ken Takata wrote:

> One more comment.
> The 'ts' option is set to 15 in this line:
> https://github.com/vim/vim/blob/b52d4b02580d20c0241c1f93d0ba47e157777980/runtime/optwin.vim#L150
> However there are some options which name is longer than 15 chars:
> toolbariconsize, balloonevalterm, quickfixtextfunc and pythonthreehome.
> 
> These options are not shown nicely in the optwin window.
> How about using the vartabstop option? E.g. `setlocal vts=15,2`
> (Or we can increase the value of 'ts' to 17, but it might break the layout of other part.)

The layout doesn't need to be perfect, so long as it's easy to read the
text.  I don't think this needs to change.

-- 
hundred-and-one symptoms of being an internet addict:
104. When people ask about the Presidential Election you ask "Which country?"

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by brammool** [comment]
Ken Takata wrote:

> > @@ -617,7 +617,7 @@ call <SID>AddOption("mousemodel", gettext("\"extend\", \"popup\" or \"popup_setp
>  call <SID>OptionG("mousem", &mousem)
>  call <SID>AddOption("mousetime", gettext("maximum time in msec to recognize a double-click"))
>  call append("$", " \tset mouset=" . &mouset)
> -call <SID>AddOption("ttymouse", gettext("\"xterm\", \"xterm2\", \"dec\" or \"netterm\"; type of mouse"))
> +call <SID>AddOption("ttymouse", gettext("\"xterm\", \"xterm2\", \"dec\", \"sgr\", etc.; type of mouse"))
> 
> I think that "sgr" should be the best nowadays if the terminal supports it.

And "dec" is probably not used very often, let's leave that one out.

> > @@ -938,7 +938,7 @@ endif
>  
>  if has("folding")
>    call <SID>Header(gettext("folding"))
> -  call <SID>AddOption("foldenable", gettext("set to display all folds open"))
> +  call <SID>AddOption("foldenable", gettext("unset to display all folds open"))
> 
> I'm not confident about this, but the meaning looks opposite.

Yes, it should be "unset". 

-- 
hundred-and-one symptoms of being an internet addict:
105. When someone asks you for your address, you tell them your URL.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by brammool** [comment]
Ken Takata wrote:

> Other thoughts:
> 
> * Support for RISC OS has been removed. Do we still need the description of the 'osfiletype' option?
> * Support for Farsi has been removed. Do we still need the description of 'altkeymap' and 'fkmap'?

These can indeed be removed.

-- 
hundred-and-one symptoms of being an internet addict:
103. When you find yourself in the "Computer" section of Barnes & Noble
     enjoying yourself.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by k-takata** [comment]
> These can indeed be removed.

Okay, 'macatsui' can also be removed.

**Comment by k-takata** [comment]
I think this is the last comment to optwin.vim.

https://github.com/vim/vim/blob/683581eb4980eda27d6438caab85c0a9cf551c24/runtime/optwin.vim#L866
'nrformats' also supports "bin" and "unsigned".

https://github.com/vim/vim/blob/683581eb4980eda27d6438caab85c0a9cf551c24/runtime/optwin.vim#L964
'foldmethod' also supports "diff".

https://github.com/vim/vim/blob/683581eb4980eda27d6438caab85c0a9cf551c24/runtime/optwin.vim#L1313
'virtualedit' also supports "onemore".

https://github.com/vim/vim/blob/683581eb4980eda27d6438caab85c0a9cf551c24/runtime/optwin.vim#L1353
'buftype' has more options.

**Comment by brammool** [comment]
> Okay, 'macatsui' can also be removed.

Yes, I'll do that.

-- 
FIXME and XXX are two common keywords used to mark broken or incomplete code
not only since XXX as a sex reference would grab everybody's attention but
simply due to the fact that Vim would highlight these words.
					-- Hendrik Scholz

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by brammool** [comment]
Ken Takata wrote:

> I think this is the last comment to optwin.vim.
> 
> https://github.com/vim/vim/blob/683581eb4980eda27d6438caab85c0a9cf551c24/runtime/optwin.vim#L866
> 'nrformats' also supports "bin" and "unsigned".
> 
> https://github.com/vim/vim/blob/683581eb4980eda27d6438caab85c0a9cf551c24/runtime/optwin.vim#L964
> 'foldmethod' also supports "diff".
> 
> https://github.com/vim/vim/blob/683581eb4980eda27d6438caab85c0a9cf551c24/runtime/optwin.vim#L1313
> 'virtualedit' also supports "onemore".
> 
> https://github.com/vim/vim/blob/683581eb4980eda27d6438caab85c0a9cf551c24/runtime/optwin.vim#L1353
> 'buftype' has more options.

I'll add the alternate values.  Some lines need to be split now.
For "buftype" I don't think it is useful to list all values, one needs
to read the explanation to understand the meaning.  I'll add "etc.".

-- 
hundred-and-one symptoms of being an internet addict:
115. You are late picking up your kid from school and try to explain
     to the teacher you were stuck in Web traffic.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Pull request closed without merging**